src/features/room/list/ui/HomeRoomStage.module.css

```
@@ -0,0 +1,143 @@
+.slot[data-slot="off-left"] {
+  --slot-left: calc(-1.1 * var(--card-side-size));
+}
+
+.slot[data-slot="off-right"] {
+  --slot-left: calc(100% + 1.1 * var(--card-side-size));
+}
+
+.slot[data-slot="current"] .slotCard {
+  --card-width: var(--card-current-width);
+  --card-height: var(--card-current-height);
+  opacity: 1;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)